sdk/cognitiveservices/ci.yml

```diff
@@ -13,18 +27,69 @@ pr:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
     - sdk/cognitiveservices/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: cognitiveservices
\ No newline at end of file
+    ServiceDirectory: cognitiveservices
+    ArtifactName: packages
+    Artifacts:
+    - name: Microsoft.Azure.CognitiveServices.AnomalyDetector
```

Comment:
Could we include the directory name in this array? That would help with Doc generation. Directory names for all other packages are named using the actual package names. Cognitive Insights is a n exception since it was shortened to help resolve long path issues. So the package directories don't have the `Microsoft.Azure.CognitiveService` Prefix

Reply:
We could do that. At the moment these values are used exclusively in the release side of the pipelines, but they could be used on the build side as well. You'd want to make it so that it defaults to name.

Reply:
Aren't we doing the doc generation as part of the build, so the ServiceDirectory would be the filter? For publishing we shouldn't need the name as we just need to get the generated artifacts.